src/options.ts

```diff
@@ -190,6 +192,13 @@ export function optionsFromParameter(parameter: string | undefined): Options {
     }
   }
 
+  if (options.enumUnspecifiedAsUndefined) {
+    // enumUnspecifiedAsUndefined requires stringEnums=true
```

Comment:
I know that, at runtime, `msg.enumValue === 0` is easy to check, but b/c during codegen time we can easily pick up the `value.number === 0` enum, i.e.:

```
      const enumProto = typeMap.get(field.typeName)![2] as EnumDescriptorProto;
      const zerothValue = enumProto.value.find((v) => v.number === 0) || enumProto.value[0];
```

It seems like we should be able to support this for all enums, and not just `stringEnums`?

Reply:
yeah, indeed that is not hard to support

Reply:
I found we don't need to do any thing to support that, fs proto3 really did not serialize default values, just change createBaseXXX and make enum field default value to undefined, the decode method wouldn't read value to message field